src/main/kotlin/apply/application/mail/MailService.kt

```diff
@@ -119,4 +111,16 @@ class MailService(
             )
         )
     }
+
+    fun createCommonMailBodyFrom(mailData: MailData): String {
```

Comment:
c: 보다 간결하고 명확한 함수 이름을 제안해 봐요!

```suggestion
    fun String.toMarkdownBody(): String {
```

Reply:
리턴 되는 문자열이 html이라 `toHtmlBody`로 변경했어요.